go.mod

```diff
@@ -3,7 +3,8 @@ module github.com/ServiceWeaver/weaver-kube
 go 1.20
 
 require (
-	github.com/ServiceWeaver/weaver v0.15.0
+	// DO_NOT_SUBMIT: Update to v0.17.0.
```

Comment:
I am a little confused by this comment.

Reply:
I'm gonna change this to v0.17.0 before submitting, but that tag doesn't exist yet. The v0.16.2-0.20230711164958-b181a1471769 points to the same commit that v0.17.0 will be.